_posts/agenda/2016-09-02-agenda.markdown

```diff
@@ -18,17 +18,19 @@ title: "Agenda"
 	- Introduce advisor(s)
 
 ### New Business: 
-	* Reports:
-		- ICC Report
-		- Treasurer Report
+
+* Reports:
+	- ICC Report
```

Comment:
I get tab happy. I put in a new pull requests. Spaces instead of tabs this time. Well, I hope so.